.github/workflows/lint-bpf-checks.yaml

```diff
@@ -111,6 +111,10 @@ jobs:
     needs: [check_changes]
     if: ${{ needs.check_changes.outputs.bpf-tree == 'true' || needs.check_changes.outputs.bpf-tests-runner == 'true' || needs.check_changes.outputs.workflow-description == 'true' }}
     name: BPF unit/integration Tests
+    env:
+      # GitHub doesn't provide a way to retrieve the name of a job, so we have
+      # to repeated it here.
+      job_name: "BPF tests"
```

Comment:
```suggestion
      job_name: "bpf_tests"
```

Reply:
You should be able to achieve the same with: `github.job`

Reply:
This is likely also why the phrase in the comment already exists in a few places in the tree. Probably the last person to try this encountered a problem, or otherwise we would be using that property:

```
$ git grep -l "GitHub doesn't provide a way"
.github/workflows/conformance-runtime.yaml
.github/workflows/integration-test.yaml
```